scout/ecm_prep.py

```diff
@@ -116,12 +120,16 @@ def __init__(self, opts):
             self.iecc_reg_map = fp.CONVERT_DATA / "geo_map" / "IECC_EMM_ColSums.txt"
             self.ba_reg_map = fp.CONVERT_DATA / "geo_map" / "BA_EMM_ColSums.txt"
             self.state_emm_map = fp.CONVERT_DATA / "geo_map" / "EMM_State_RowSums.txt"
+            self.tsv_load_data = fp.TSV_DATA / "tsv_load.gz"
+            # self.tsv_load_data = fp.TSV_DATA / "tsv_load_emm.gz"  # UNCOMMENT
```

Comment:
Does this commented line and the one below need to remain?

Reply:
This was addressed in commit [24c7faf](https://github.com/trynthink/scout/pull/403/commits/24c7faf8750edb59600bf5442aba659ac10b8661)